configure

```diff
@@ -92,6 +92,29 @@ echo
 echo "Configuration successful"
 echo
 
+# jbuilder config file generation
```

Comment:
I think this approach is the worst of all worlds however :-) Generating jbuild files at configuration time means that build composition doesn't work, and incremental builds also suffer.  Using the configurator library gets around that by making the flag probing part of the build itself.